website/www/site/assets/js/fix-playground-nested-scroll.js

```diff
@@ -10,10 +10,10 @@
 // License for the specific language governing permissions and limitations under
 // the License.
 
-$('#playgroundIframeContainer').on('click', function(_) {
-  $('#playgroundIframe').css('pointer-events', 'auto');
+$('.playgroundIframeContainer').on('click', function(_) {
+  $('.playgroundIframeContainer > iframe').css('pointer-events', 'auto');
```

Comment:
I think it's better to add and remove classes to the div instead if tweaking styles at runtime. I suggest to:
1. Rename this class to `playground-iframe-wrapper-no-scroll` so it is self-descriptive. (Dashes are for this to go in line with my parallel work on embedding).
2. On click, add another class `playground-iframe-wrapper-scroll` that sets `pointer-events: auto` on iframe so it takes over.
3. On mouse leave, remove `playground-iframe-wrapper-scroll` class.

Reply:
done

Reply:
done